### PR TITLE
Added associates CSV upload

### PR DESCRIPTION
**Pull request by dusanradovanovic**
Description:
This PR adds ability to upload multiple associates using the CSV file.

Linting:

- [x] No linting errors

Component Updates:

- [ ] Component package requires update on npm [@elasticpath/store-components](https://www.npmjs.com/package/@elasticpath/store-components)

Tests:

- [ ] E2E tests (npm test run with `e2e`)
- [x] Manual tests

Documentation:

- [ ] Requires documentation updates

### CODE REVIEW

File: app/src/localization/fr-FR.json [hunk 1]
```diff
@@ -211,7 +211,7 @@
   "email-slash-username": "E-m-a-i-l-/-U-s-e-r-n-a-m-e",
   "password-confirmation": "P-a-s-s-w-o-r-d- -C-o-n-f-i-r-m-a-t-i-o-n",
   "search-results": "S-e-a-r-c-h- -R-e-s-u-l-t-s",
-  "search-results-for": "S-e-a-r-c-h- -r-e-s-u-l-t-s- -f-o-r-:- -'{searchKeywords}'",
+  "search-results-for": "S-e-a-r-c-h- -r-e-s-u-l-t-s- -f-o-r-:- -'-{-s-e-a-r-c-h-K-e-y-w-o-r-d-s-}-'",
```
Comment (petro97) [inline app/src/localization/fr-FR.json]: @dusanradovanovic It breaks a variable in the text, need to update translate script.

File: app/src/containers/b2b/B2BMain.tsx [hunk 2]
```diff
@@ -35,51 +37,170 @@ interface DashboardProps {
...
+        </div>
         <div className="container">
           <div className="b2b-header">
             <div className="page-title">{intl.get('business-account')}</div>
-            <div className="quick-menu">
-              {/* <AddAssociatesMenu onSpreeadsheetClicked={() => this.handleSpreeadsheetClicked()} /> */}
-            </div>
+            {associatesFormUrl && (
+              <div className="quick-menu">
+                <AddAssociatesMenu onSpreeadsheetClicked={() => this.handleSpreeadsheetClicked()} />
```
Comment (shaunmaharaj) [inline app/src/containers/b2b/B2BMain.tsx]: Typo: onSpreeadsheetClicked

**Comment by shaunmaharaj** [comment]
@dusanradovanovic I'll merge this one now so @petro97 can make updates to his for the cleanup, then we can add any updates to both in subsequent PRs. Thanks for getting this one in!

**Pull request merged**